.github/workflows/test.yml

```diff
@@ -32,25 +32,19 @@ jobs:
     strategy:
       matrix:
         toxenv:
-            - py36-dj22-wag27
-            - py36-dj22-waglatest
-            - py36-dj31-waglatest
-            - py38-dj22-wag27
-            - py38-dj22-waglatest
+            - py37-dj30-wag215
+            - py37-dj30-waglatest
+            - py38-dj31-wag215
             - py38-dj31-waglatest
         include:
```

Comment:
I can't see the tests being run here but py37 might have some issues with Django4.0, because Django 4.0 supports Python 3.8, 3.9, and 3.10. 
If that happened ( which happened to me) , you can use `exclude` syntax here